salt/modules/boto3_sns.py

```diff
@@ -301,7 +302,7 @@ def set_subscription_attributes(SubscriptionArn, AttributeName, AttributeValue,
         return False
 
 
-def subscribe(TopicArn, Protocol, Endpoint, region=None, key=None, keyid=None, profile=None):
+def subscribe(TopicArn, Protocol, Endpoint, region=None, key=None, keyid=None, profile=None, **kwargs):
```

Comment:
So since we're going to remove these in the future, shouldn't we make these optional now? Then check for their existence below?

Reply:
I can do that :)